src/main/java/org/jenkinsci/backend/confluence/pageremover/PageNotification.java

```diff
@@ -17,22 +19,26 @@
     /**
      * "added", "edited", etc.
      */
-    public final String action;
+    public String action;
```

Comment:
Because I had to add a no-args constructor for Jackson deserialization, which didn't play nice with `final` fields.